odxtools/minmaxlengthtype.py

```diff
@@ -34,113 +34,136 @@ def __post_init__(self):
     def dct_type(self) -> DctType:
         return "MIN-MAX-LENGTH-TYPE"
 
-    def __termination_character(self):
-        """Returns the termination character or None if it isn't defined."""
-        # The termination character is actually not specified by ASAM
+    def __termination_sequence(self) -> bytes:
+        """Returns the termination byte sequence if it isn't defined."""
+        # The termination sequence is actually not specified by ASAM
         # for A_BYTEFIELD but I assume it is only one byte.
-        termination_char = None
+        termination_sequence = b''
         if self.termination == "ZERO":
             if self.base_data_type not in [DataType.A_UNICODE2STRING]:
-                termination_char = bytes([0x0])
+                termination_sequence = bytes([0x0])
             else:
-                termination_char = bytes([0x0, 0x0])
+                termination_sequence = bytes([0x0, 0x0])
         elif self.termination == "HEX-FF":
             if self.base_data_type not in [DataType.A_UNICODE2STRING]:
-                termination_char = bytes([0xFF])
+                termination_sequence = bytes([0xFF])
             else:
-                termination_char = bytes([0xFF, 0xFF])
-        return termination_char
+                termination_sequence = bytes([0xFF, 0xFF])
+        return termination_sequence
```

Comment:
Just a proposal to simplify the logic (existing logic is correct, and you may keep it as is)

```python
if self.termination == "ZERO":
    termination_sequence = bytes([0x0])
elif self.termination == "HEX-FF":
    termination_sequence = bytes([0xFF])
else:
    termination_sequence = b''
if self.base_data_type == DataType.A_UNICODE2STRING:
    termination_sequence *= 2
return termination_sequence
```

Reply:
I like the current code slightly better, even though it is a bit longer: I don't like doing arithmetic on non-numeric variables. (I'm less opposed to doing this to literals a la `[0]*num_bytes`)

Reply:
`termination_sequence = termination_sequence + termination_sequence`

Reply:
better, but I still slightly prefer the jungle of `if`s. That said, if you have more than just a cursory preference, I'll change it...

Reply:
This a proposal, you can ignore

Reply:
I only want to ignore it if you deem it to be unimportant :)

Reply:
already did, PR approved